forms-flow-components/src/components/CustomComponents/Tabs.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Tab from "react-bootstrap/Tab";
+import Tabs from "react-bootstrap/Tabs";
+
+interface TabItem {
+  eventKey: string;
+  title: string;
+  content: React.ReactNode;
+}
+
+interface CustomTabsProps {
+  defaultActiveKey: string;
+  id?: string;
+  tabs: TabItem[];
+  dataTestid?: string;
+  ariaLabel?: string;
+  onSelect?: (eventKey: string | null) => void;
+}
+
+export const CustomTabs: React.FC<CustomTabsProps> = ({
+  defaultActiveKey,
+  id = "custom-tab",
+  tabs,
+  dataTestid = "",
+  ariaLabel = "",
+  onSelect,
+}) => {
+  return (
+    <Tabs
+      defaultActiveKey={defaultActiveKey}
+      id={id}
+      className="custom-tabs mb-3"
+      data-testid={dataTestid}
+      aria-label={ariaLabel}
+      onSelect={onSelect}
+    >
+      {tabs.map((tab, index) => (
+        <Tab key={index} eventKey={tab.eventKey} title={tab.title}>
```

Comment:
Please add a data-testid attribute on this <Tab> : 
data-testid={`${dataTestid}-tab-${tab.eventKey}`}

Reply:
Updated